readme.md

```diff
@@ -111,6 +111,79 @@ python3 -m nwm_routing -f -V4 test_AnA_V4_HYFeature.yaml
 
 **Note**: The following instructions are for setting up T-Route on a Linux environment (standalone, no MPI). If you are using Windows, please install WSL (Windows Subsystem for Linux) before proceeding.
 
+### T-Route Setup and Testing Guide for Windows Users WITHOUT conda [based on pip and venv - only widely available dependencies]
+
+1. **Install Recommended distro:**
+   - Download and install WSL2 for your Windows OS
+   - We recommend long-term stable (LTS) Ubuntu distribution 22.04 or 20.04 (24.04 not recommended yet)
+   - Open Windows Power Shell and issue
+     > wsl --install Ubuntu-22.04
+   - Enter (root) username and password (2x) of your choice
+     
+2. **Set up venv-based virtual environment:**
+   - From root (the username you created under 1):
+      - Update Linux distro:
```

Comment:
Just to make this a little easier, I would recommend creating a shell formatted code block (e.g. ```shell) and use comments instead of a unordered list (like below). GitHub adds a copy button to the top right of these blocks and as a user that is super nice rather than having to go back and forth copy and pasting.

```shell
# Update Linux distro:
sudo apt update

# Install pip (package manager):
sudo apt install python3-pip

# Install venv:
sudo apt install python3.10-venv
```

Aside, I would move the step for creating the virtual environment after `t-route` has been cloned and that the environment be created in the clone. Just a recommendation though!

Reply:
Point well taken on the formatted code block. 

Regarding the order of cloning and creating the virtual environment: the steps are meant to replicate the previous procedure based on the conda environment, which (I think) should be kept in the instructions as an alternative (many users have existing conda environments, and we should give them the choice whether they want to move to pip). In that sense, I think I'll opt for starting with venv.

Reply:
Offline discussion on the virtual environment - the rationale for Inland Hydraulics to use a virtual environment as an umbrella within which to clone t-route and related code is that t-route works closely with auxiliary tools such as those found in the troute-tools repository https://github.com/JurgenZach-NOAA/t-route_tools, and we need to ensure compatibility, so they always live within the same virtual environment.